rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -275,18 +258,14 @@ def _process_tokens(self, tokens: List[Text], attribute: Text = TEXT) -> List[Te
     def _replace_with_oov_token(
         self, tokens: List[Text], attribute: Text
     ) -> List[Text]:
-        """Replace OOV words with OOV token"""
-
+        """Replace OOV words with OOV token."""
         if self.OOV_token and self.analyzer == "word":
-            vocabulary_exists = self._check_attribute_vocabulary(attribute)
-            if vocabulary_exists and self.OOV_token in self._get_attribute_vocabulary(
-                attribute
-            ):
+            attribute_vocab = self._get_attribute_vocabulary(attribute)
+            if attribute_vocab is not None and self.OOV_token in attribute_vocab:
                 # CountVectorizer is trained, process for prediction
+                attribute_vocabulary_tokens = list(attribute_vocab.keys())
```

Comment:
we can skip the `list` call, no?
```suggestion
                attribute_vocabulary_tokens = attribute_vocab.keys()
```

Reply:
hm not sure what the runtime of element search is in the `dict_keys` class, but might actually make sense to change it to set!?